libcxx/test/std/utilities/intseq/intseq.binding/integer_seq.pass.cpp

```diff
@@ -0,0 +1,48 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: std-at-least-c++26
+// <utility>
```

Comment:
```suggestion
// REQUIRES: std-at-least-c++26

// <utility>
```
nit: an empty line for readability.